docs/modules/ROOT/pages/vshn-managed/postgresql/restore.adoc

```diff
@@ -64,3 +64,183 @@ The only difference is the backup information in `spec.parameters.restore`.
 
 The deletion process of a restored instance is the same as for normal instance.
 Check out xref:vshn-managed/postgresql/delete.adoc[this guide] how to delete a PostgreSQL instance.
+
+== Restore a single database
+
+This procedure allows you to restore a specific database from a backup to a running VSHNPostgreSQL instance. This is useful when you only need to recover one database instead of restoring an entire instance.
```

Comment:
It is important to mention that you need to specify a targetTimeStamp if you want an older backup:

```
    restore:
      claimName: vshn-postgres-int
      backupName: vshn-postgres-int-jmjpn-2025-12-10-01-53-02
      recoveryTimeStamp: 2025-12-10T08:00:19+00:00
```
BackupName: time of the backup has to be before the recoveryTimeStamp

+ mention again that the new claim has to be in the same namespace as the old one

If you don't specify recoveryTimeStamp, it recovers to the newest available timestamp in the wal log...which might be now.